src/plugins/intel_gpu/include/intel_gpu/primitives/lstm.hpp

```diff
@@ -145,4 +145,160 @@ struct lstm_elt : public primitive_base<lstm_elt> {
     }
 };
 
+
+struct lstm_seq : public primitive_base<lstm_seq> {
```

Comment:
I suggest making it more generic and support different rnn kinds within 1 primitive

Reply:
updated